samples/XCT.Sample/ViewModels/Converters/IsNullOrEmptyConverterViewModel.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.ObjectModel;
+using System.Windows.Input;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.Sample.ViewModels.Converters
+{
+	public class IsNullOrEmptyConverterViewModel : BaseViewModel
+	{
+		public ObservableCollection<string> DummyItemSource { get; set; } = new ObservableCollection<string>
+		{
+			"Dummy Item 0",
+			"Dummy Item 1",
+			"Dummy Item 2",
+			"Dummy Item 3",
+			"Dummy Item 4",
+			"Dummy Item 5",
+		};
+
+		string selectedItem;
+		public string SelectedItem
+		{
+			get => selectedItem;
+			set
+			{
+				selectedItem = value;
+				OnPropertyChanged();
+			}
+		}
+
+		public ICommand ClearSelectionCommand => new Command(() =>
+		{
+			SelectedItem = null;
+		});
```

Comment:
```suggestion
		ICommand clearSelectionCommand;
		public ICommand ClearSelectionCommand => clearSelectionCommand ??= new Command(() =>
		{
			SelectedItem = null;
		});
```

Reply:
I knew "??" but not "??=" good to know about 👍🏻